shotgun_api3/shotgun.py

```diff
@@ -224,10 +224,10 @@ def __init__(self, host, meta):
         except AttributeError:
             self.version = None
         if not self.version:
-            raise ShotgunError("The ShotGrid Server didn't respond with a version number. "
+            raise ShotgunError("The Flow Production Tracking Server didn't respond with a version number. "
                                "This may be because you are running an older version of "
-                               "ShotGrid against a more recent version of the ShotGrid API. "
-                               "For more information, please contact ShotGrid Support.")
+                               "Flow Production Tracking against a more recent version of the Flow Production Tracking API. "
+                               "For more information, please contact Flow Production Tracking Support.")
```

Comment:
I think for support, we should refer to Autodesk instead?

```suggestion
                               "For more information, please contact the Autodesk support.")
```